homework6.py

```diff
@@ -0,0 +1,36 @@
+"""Homework 6 14.05.2024
+Murashko Maksim Yurevich
+Python automatic tester course"""
+
+# Перевести строку в список "Robin Singh" => ["Robin”, “Singh"]
+A = "Robin Singh"
```

Comment:
please read this PEP about naming: https://peps.python.org/pep-0008/#descriptive-naming-styles